src/lib/webtab/webtab.cpp

```diff
@@ -385,6 +385,10 @@ void WebTab::reload()
 
 void WebTab::load(const LoadRequest &request)
 {
+    if (!isRestored()) {
+        tabActivated();
+        QTimer::singleShot(300, this, std::bind(&WebTab::load, this, request));
```

Comment:
Can you try only with QTimer::singleShot(0)?

Although maybe clean solution is to just remove the singleShot(0) call from tabActivated, I'm not sure if it really is needed at all there.

Reply:
yeah... it worked with singleShot(0)